spec/azure/configuration_spec.rb

```diff
@@ -6,16 +6,4 @@
       JSON.parse(File.read('azure/template.json'))
     end
   end
-
-  describe 'azure-pipelines.yml' do
```

Comment:
I'm not sure the space saving from deleting our pipeline files from the image is worth the cost of risking broken pipelines in the future?

Reply:
this was not for space saving, feel this test is redundant and the tests run inside the container and the container wouldn't have been built if the yaml was not proper, further the file could still be a valid yaml but divert away from the pipeline schema.

Reply:
We do sometimes change the files and run specs outside the container, eg when we're adding ENV vars, so I think this still has some value. Separately, it might be interesting to run what we've got through https://github.com/microsoft/azure-pipelines-vscode/blob/master/service-schema.json.

Reply:
not ignoring yml files and adding valid YAML test for all pipeline files.